### PR TITLE
Notebook 1b final fixes

### PR DESCRIPTION
**Pull request by IbrahimKhan07**

### CODE REVIEW

File: staticWebsite/content/appointments_attended.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Understanding the frequency of missed appointments in SNEE footprints is crucial
 
 
 # Data Sources
-This analysis draws its primary data from the comprehensive appointments dataset provided by NHS England. The dataset encompasses a substantial time span, ranging from April 2021 to August 2023  
+This analysis draws its primary data from the comprehensive appointments dataset provided by NHS England. The dataset encompasses a substantial time span, ranging from April 2021 to August 2023. FY 2021/2022 AND FOR FY23 onlyt ill august/23
```
Comment (AJarman) [inline staticWebsite/content/appointments_attended.md:16]: Correct typo/ sentence please

File: staticWebsite/content/appointments_attended.md [hunk 2]
```diff
@@ -43,10 +43,16 @@ The proportion was calculated by dividing the count of DNA/Attended appointments
...
+  <tbody><tr><td>2021</td><td>579,332.27</td><td>450,111.80</td><td>27,252.80</td><td>16,323.20</td></tr><tr><td>2022</td><td>608,099.22</td><td>465,021.90</td><td>31,568.70</td><td>17,822.60</td></tr><tr><td>2023</td><td>591,085.91</td><td>448,269.20</td><td>29,004.74</td><td>14,968.00</td></tr></tbody>
+</table>
+
+
+From the above table we can easily infer AVERAGE FOR EACH MONTH
 
-- The average count of non-attended appointments for SNEE-ICB stands notably lower at 101, distinctly below the National Average of 304.
-- Noteworthy is the trajectory in 2022, where this variance intensifies. The National Average escalates from 304 to 334, a significant uptick, while the SNEE-ICB sees a comparably modest shift from 101 to 106.  
+- The average count of non-attended appointments for SNEE-ICB stands notably lower, distinctly below the National Average for each year.
+- Noteworthy is the trajectory in 2022, where this variance intensifies. The National Average escalates from  579,332.27 to  608,099.22, a significant uptick, while the SNEE-ICB sees a comparably modest shift from  16,323.20 to  17,822.60.
```
Comment (AJarman) [inline staticWebsite/content/appointments_attended.md:55]: Can we remove this please and just focus on rate of appointments missed

**Review by AJarman — CHANGES_REQUESTED** [review]
Please review comments and and revise markdown file.
Thanks
Andrew

File: staticWebsite/content/appointments_attended.md (hunk 1 above)
Comment (IbrahimKhan07) [reply staticWebsite/content/appointments_attended.md]: resolved

File: staticWebsite/content/appointments_attended.md (hunk 2 above)
Comment (IbrahimKhan07) [reply staticWebsite/content/appointments_attended.md:55]: Just keeping the table as placeholder, to replace the rates with proportions in the next PR.

**Pull request merged**